lib/network/modules/components/shared/Label.js

```diff
@@ -127,7 +127,13 @@ class Label {
     // configure context for drawing the text
     ctx.font = (selected && this.nodeOptions.labelHighlightBold ? 'bold ' : '') + fontSize + "px " + this.fontOptions.face;
     ctx.fillStyle = fontColor;
-    ctx.textAlign = 'center';
+    // When the textAlign property is 'left', make label left-justified
+    if (this.options.font.textAlign === 'left') {
+      ctx.textAlign = this.options.font.textAlign;
+      x = x - (this.size.wideth >> 1); // Shift label 1/2-way (>>1 == div by 2) left
```

Comment:
Hah. Will fix, yep - typo. As for the shift, sure - * 0.5 is probably close to similar efficiency. The C part of my brain automatically translates x / 2^n -> (x >> n), but IIRC multiply-by-coefficient is also much faster than div.